File: java/Ram.java
package latihan3.java;

public class Ram {

    private int capacity;
    private int price;

    public Ram(int capacity, int price) {
        this.capacity = capacity;
        this.price = price;
    }

    public Ram() {
    }

    public int getPrice() {
        return price;
    }

    public int getCapacity() {
        return capacity;
    }

    public void setCapacity(int capacity) {
        this.capacity = capacity;
    }

    public void setPrice(int price) {
        this.price = price;
    }

}

File: cpp/Ram.cpp
#include <bits/stdc++.h>

using namespace std;

class Ram
{
private:
    int capacity, price;

public:
    Ram()
    {

        this->capacity = 0;
        this->price = 0;
    }

    Ram(int capacity, int price)
    {

        this->capacity = capacity;
        this->price = price;
    }

    void setCapacity(int capacity)
    {
        this->capacity = capacity;
    }

    void setPrice(int price)
    {
        this->price = price;
    }

    int getCapacity()
    {
        return this->capacity;
    }

    int getPrice()
    {
        return this->price;
    }

    ~Ram(){};
};
